src/Controller/OrganizationController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\CommissionableAsset;
+use App\Entity\Organization;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/organization")
+ */
+class OrganizationController extends AbstractController
+{
+    /**
+     * @Route("/home", name="organization_home", methods={"GET", "POST"})
+     */
+    public function home(Request $request): Response
+    {
+        $om = $this->getDoctrine()->getManagerForClass(Organization::class);
+        $organization = $om->find(Organization::class, 1);
+
+        return $this->render('organization/home.html.twig', [
+            'organization' => $organization
+        ]);
+    }
+
+    /**
+     * @Route("/commissionables", name="organization_commissionables", methods={"GET", "POST"})
+     */
+    public function commissionables(Request $request): Response
+    {
+        $om = $this->getDoctrine()->getManagerForClass(Organization::class);
+        $organization = $om->find(Organization::class, 1);
+
+        $assets = $om->getRepository(CommissionableAsset::class)->findByOrganization($organization);
```

Comment:
:warning: I think there is an error here

```suggestion
        $assetRepository = $this->getDoctrine()->getManagerForClass(CommissionableAsset::class);
        $assets = $assetRepository->findByOrganization($organization);
```